File: BackEnd/WebApiSolveBank/SolveBank.Entities/Models/ContaBancaria.cs
using SolveBank.Entities.Enums;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;


namespace SolveBank.Entities.Models
{
    public class ContaBancaria
    {
        [Key]
        public Guid Id { get; set; }
        public string Agencia { get; set; } = null!;
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]        
        public int Numero { get; set; }
        public decimal Saldo { get; set; }
        public decimal Limite { get; set; }
        public virtual List<Transacao> Transacoes { get; set; }
        [Required]
        [ForeignKey("Usuario")]
        public string UsuarioID { get; set; } = null!;
        public virtual Usuario Usuario { get; set; } = null!;
        public List<Cartao> Cartoes { get; set; }
        public List<Atendimento> Atendimentos { get; set; }
        public EnumCategoriaConta EnumCategoriaConta { get; set; }
        public string Informacoes { get; set; } = null!;
        public bool Removido { get; set; }
    }
}

File: FrontEnd/src/app/models/ContaBancaria/ContaBancaria.ts
import { Atendimento } from "../Atendimento/Atendimento"
import { Cartao } from "../Cartao/Cartao"
import { Transacao } from "../Transacoes/Transacao"

export class ContaBancaria {

    Id: string = ""
    agencia: string = ""
    numero: number = 0
    saldo: number = 0
    limite: number = 0
    limiteUtilizado: number = 0
    transacoes: Transacao[] = []
    usuarioID: string = ""
    Cartoes: Cartao[] = []
    Atendimentos: Atendimento[] = []
    EnumCategoriaConta: number = 0
    Informacoes: string = ""
    removido: boolean = false

}